_posts/2025-29-01-indico-global-launched.md

```diff
@@ -0,0 +1,25 @@
+---
+layout:     post
+title:      "indico.global is live!"
+date:       2025-29-01 17:00
+author:     "Indico Team"
+categories: indico global
+---
+
+We are happy to announce that [indico.global](indico.global) is now available to
```

Comment:
```suggestion
We are happy to announce that [indico.global](https://indico.global) is now available to
```

I would remove the links from the other occurrences below, one is probably enough...